website/common/locales/en/messages.json

```diff
@@ -59,5 +59,7 @@
 
   "messageUserOperationProtected": "path `<%= operation %>` was not saved, as it's a protected path.",
   "messageUserOperationNotFound": "<%= operation %> operation not found",
-  "messageNotificationNotFound": "Notification not found."
+  "messageNotificationNotFound": "Notification not found.",
+
+  "bannedWordUsed": "You can not use banned words in chat."
```

Comment:
Also, maybe specify that this only applies for public (non-private) chat? @Alys?